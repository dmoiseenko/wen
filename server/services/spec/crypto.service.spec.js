const cryptoService = require('../crypto.service.js');


it('should encrypt data', () => {
  const data = 'data';

  const encrypted = cryptoService.encrypt(data);

  expect(encrypted).toMatchSnapshot();
});

it('should decrypt data', () => {
  const data = '8b75241f7286cea1d406a2fd34d625ee';

  const actual = cryptoService.decrypt(data);

  expect(actual).toMatchSnapshot();
});
