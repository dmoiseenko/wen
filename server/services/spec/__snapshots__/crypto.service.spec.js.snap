// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should decrypt data 1`] = `"�#��L��0�Yз����"`;

exports[`should encrypt data 1`] = `"4237cc90"`;
